conda_smithy/configure_feedstock.py

```diff
@@ -571,7 +574,96 @@ def _load_forge_config(forge_dir, variant_config_files):
     return config
 
 
-def main(forge_file_directory, variant_config_files):
+def get_installed_version(pkg):
+    from conda_build.conda_interface import root_dir, linked
+    root_linked = linked(root_dir)
+    vers_inst = [dist.split('::', 1)[-1].rsplit('-', 2)[1] for dist in root_linked
+        if dist.split('::', 1)[-1].rsplit('-', 2)[0] == pkg]
+    if not len(vers_inst) == 1:
+        return None
+    return vers_inst[0]
```

Comment:
@msarahan, I borrowed above from conda-build. I'll add the license

Reply:
OK - or just let me know which function you borrowed from and we can talk about exposing it somehow.  Since this code is somewhat reliant on assumptions in conda, it might be good to have it only defined in conda-build.

Reply:
Got it from here, https://github.com/conda/conda-build/blob/2.1.x/conda_build/build.py#L745

Reply:
@msarahan, is there a better way to do this using conda or conda-build?

Reply:
Note that I generalized it a little bit.  It expects a prefix (not always root), and it accepts either a name of a package, or an iterable collection of names of packages.

Reply:
Thanks. I'll update this PR once there's a new conda-build release